demo/Makefile

```diff
@@ -53,9 +53,14 @@ dynamic: easywsclient.o
 		$(FLAGS) \
 		-L lib \
 
-x86_64_build: prepare_x86_64 easywsclient.o.x86_64
+x86_64_run: x86_64_build
+	export DYLD_LIBRARY_PATH=$(PWD)/lib && ./demo
```

Comment:
Could this be tested in CI?

Reply:
I agree dylib should be copied into @rpath. We build in g++, unfortunately it is not an app bundle. Override  DYLD_LIBRARY_PATH seems can not verify `rpath` works. So how about setup `LD_RUNPATH_SEARCH_PATH` (not tested) instead?

Reply:
ok, i'll add to mac-build.yml , checking whether dynamic dylib can be called correctly

Reply:
Tested `LD_RUNPATH_SEARCH_PATH`, can not work..

Reply:
Passed `-rpath lib` to g++ can fix the issue. No need to export env.

Reply:
So unreal and g++ both use the same rpath setting like you mentioned before.

```
 otool -L libplay_cpp_sdk.dylib
libplay_cpp_sdk.dylib:
	@rpath/libplay_cpp_sdk.dylib (compatibility version 0.0.0, current version 0.0.0)
```

Reply:
add rlib option and also comment, too.